vera/README.md

```
@@ -85,7 +85,7 @@ execute(compile_result, ...)    # → run WASM via wasmtime
 | `  eq_ability.py` | 199 | | Eq ability derivation checks | |
 | `  sql.py` | 308 | | SQL literal-provenance resolution + placeholder counting (#309) | `resolve_literal_string()`, `count_placeholders()` |
 | `  calls.py` | 1,556 | | Function/constructor/module/ability calls | |
-| `  control.py` | 627 | | If/match, patterns, effect handlers | |
+| `  control.py` | 709 | | If/match, patterns, effect handlers | |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)